packages/datadog-plugin-promise-js/src/index.js

```diff
@@ -5,7 +5,21 @@ const tx = require('../../dd-trace/src/plugins/util/promise')
 module.exports = [
   {
     name: 'promise-js',
-    versions: ['>=0.0.3'],
+    versions: ['>=0.0.7'],
```

Comment:
Not needed since the native promise will always be returned for Node.

Reply:
Ok, makes sense, removing the multiple version thing and just keeping the defensive check on native promises to avoid patching a second time